src/app/components/Modal/Modal.tsx

```diff
@@ -18,14 +19,33 @@ type ModalContentProps = {
 	title: string;
 	description?: string;
 	image?: React.ReactNode;
+	size?: "sm" | "md" | "lg" | "xl" | "default";
```

Comment:
Could we completely omit the `"default"` value and just allow null? Since we have the `default` case it seems redundant to keep this value.